shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4

```diff
@@ -234,8 +234,8 @@ showStatus
     : SHOW (GLOBAL | SESSION)? STATUS showFilter?
     ;
 
-showTrriggers
-    : SHOW TRIGGER fromSchema? showFilter?
+showTriggers
+    : SHOW TRIGGERS fromSchema? showFilter?
```

Comment:
@Swastyy Can you add some parse test for this change?

Reply:
Is there any example PR which can be a reference for it?

Reply:
> Is there any example PR which can be a reference for it?

@Swastyy You can refer pr #13446.